test-suite/start-test-environment.sh

```diff
@@ -40,11 +40,26 @@ docker-compose -f docker-compose.yml up -d activemq redis dynamodb ps_db outboun
 docker-compose -f docker-compose.yml up db_migration
```

Comment:
Also, there is a similar file start-local-environment.sh stored under "docker" folder which might benefit from the same sort of changes that we are doing in this file. I think we should also adjust that file as well.

Reply:
Good suggestion. Is it a blocker for this improvement though?

Reply:
no, it is not and can be done separately...